syntax-tools/sublime-text/dragon.sublime-syntax

```diff
@@ -26,7 +26,7 @@ contexts:
     # strings in YAML. When using single quoted strings, only single quotes
     # need to be escaped: this is done by using two single quotes next to each
     # other.
-    - match: '\b(if|else|for|while|loop|fn|struct|return)\b'
+    - match: '\b(if|else|for|while|loop|fn|struct|return|break|continue)\b'
```

Comment:
```in``` keyword missing ?

Reply:
too minor for a commit i suppose. the lang will probably change several times in the future, so lets postpone a commit for this.